src/main/features/directions-questionnaire/views/expert-reports.njk

```diff
@@ -0,0 +1,68 @@
+{% extends "layout.njk" %}
+{% from "form.njk" import csrfProtection, radioOption, linkButton, saveAndContinueButton, errorSummary, textInput %}
+{% from "date.njk" import dateInput %}
+
+{% set heading = 'Have you already got a report written by an expert?' %}
+
+{% macro expertReportsRow(index, form) %}
+
+  {% set prefix = 'rows[' + index + ']' %}
+  {% set nameKey = prefix + '[expertName]' %}
+  {% set dateKey = prefix + '[reportDate]' %}
+
+  <div class="form-group {% if form.errorFor(nameKey) or form.errorFor(dateKey) %} form-group-error {% endif %}">
+
+    {{ textInput('Expert’s name', name = nameKey, form = form) }}
+    {{ dateInput(dateKey, form, 'When is the report dated?', 'For example, ' + ('now' | addDays(-30) | date)) }}
+
+  </div>
+
+{% endmacro %}
+
+{% block content %}
+  <div class="grid-row">
+    <div class="column-two-thirds">
+
+      <form method="post" novalidate
+            class="analytics-submit-event-trigger"
+            data-event-action="DQ: Have you already got a report written by an expert?"
+            data-event-label-from="declared">
+
+        {{ csrfProtection(csrf) }}
+
+        {% set errorHasAnyExpertReports = form.errorFor('declared') %}
+        <div class="statement-of-means-container">
```

Comment:
this class needs to be renamed either here or in the scss file to be more generic

Reply:
The div itself seems unnecessary here. Will remove.
(Copy and paste misdemeanour of course 👎 )